src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -584,7 +545,8 @@ def build(self, notification_uuid: str | None = None) -> SlackBlock:
         # If an event is unspecified, use the tags of the latest event (if one exists).
         event_for_tags = self.event or self.group.get_latest_event()
 
-        obj = self.event if self.event is not None else self.group
+        event_or_group: Group | GroupEvent = self.event if self.event is not None else self.group
```

Comment:
I don't think this should need an annotation?  mypy should be able to infer this union I think

Reply:
it gets angry if i take it away

Reply:
what's the error? -- does it narrow it to `object` (probably)? :(

Reply:
yes